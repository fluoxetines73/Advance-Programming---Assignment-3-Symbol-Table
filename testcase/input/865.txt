BEGIN
INSERT f_FF number
BEGIN
LOOKUP f_FF
RPRINT
BEGIN
ASSIGN f_FF 789 random
ASSIGN f_FF 456
RPRINT
PRINT
ASSIGN f_FF 789
PRINT
ASSIGN f_FF 123
END
END
END