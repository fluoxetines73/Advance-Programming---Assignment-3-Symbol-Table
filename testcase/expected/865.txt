Invalid: ASSIGN f_FF 789 random